pkgs/development/python-modules/poetry/default.nix

```diff
@@ -82,6 +78,7 @@ buildPythonPackage rec {
     requests-toolbelt
     shellingham
     tomlkit
+    trove-classifiers
```

Comment:
From https://github.com/python-poetry/poetry/commit/b28339d8b76500025fadcd23c87debfc368b609c, I think tomli is needed in Python < 3.11.